llvm/test/CodeGen/RISCV/GlobalISel/float-fclass.ll

```diff
@@ -0,0 +1,21 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+
+; RUN: llc -mtriple=riscv32 -mattr=+f -global-isel \
+; RUN:   -disable-gisel-legality-check -verify-machineinstrs \
```

Comment:
Can you drop the -disable-gisel-legality-check and -verify-machineinstrs.

Reply:
Yeah no problem :)